docs/source/standard.md

```diff
@@ -39,67 +39,92 @@ What the TRE organisation does to ensure information risk is measured and manage
 
 ### 1.6 Project management
 
-The ability of the TRE organisation to manage projects effectively.
+*The ability of the TRE organisation to manage projects effectively.*
 
-### 1.7 Researcher accreditation
+#### Project onboarding
 
-The ability of the TRE organisation to ensure that people with access to data are identified correctly and they are suitably qualified.
+| Statement | Guidance | Mandatory status |
+| --------- | -------- | ---------------- |
+| You have checks in place to ensure a project has the legal, financial and ethical requirements in place for the duration of the project | This includes checks that contracts are in place where required, adequate funding is available for the duration of the project, and responsibilities concerning data ownership are understood by all parties | Mandatory |
 
-#### 1.7.1 Training management
+#### Project closure
 
-The ability of the TRE organisation to track and maintain adequate training levels.
+| Statement | Guidance | Mandatory status |
+| --------- | -------- | ---------------- |
+| You have standard processes in place for the end of a project, that follow all legal requirements and data security best practice | This includes the archiving of quality and log data along with the archiving or deletion of data sets | Mandatory |
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+#### Roles and responsibilities
 
-#### 1.7.2 Training delivery
+| Statement | Guidance | Mandatory status |
+| --------- | -------- | ---------------- |
+| You have clearly defined roles and responsibilities within your TRE organisation for all members | This may include roles such as users, system administrators, system operators, data providers and more. Every member of your TRE organisation should have a pre-defined role with clear powers and responsibilities | Mandatory |
 
-The ability of the TRE organisation to deliver training.
+### 1.7 Researcher accreditation
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+The ability of the TRE organisation to ensure that people with access to data are identified correctly and they are suitably qualified.
+
+#### Onboarding members
+
+| Statement | Guidance | Mandatory status |
+| --------- | -------- | ---------------- |
+| You have clear onboarding processes in place for all roles within your TRE organisation | This may include all members signing role-specific terms of use, and completing role specific training | Mandatory
+| You have a robust method for identifying accredited members of your TRE organisation, prior to their accessing of sensitive data | This may include 2 factor authentication, ID checks or email verification | Mandatory |
```

Comment:
```suggestion
| You have a robust method for identifying accredited members of your TRE organisation, prior to their accessing of sensitive data | This may include multi-factor authentication (MFA), ID checks or email/phone verification | Mandatory |
```